src/main/java/org/spongepowered/common/event/tracking/PhaseContext.java

```diff
@@ -98,6 +99,13 @@ public PhaseContext add(@Nullable NamedCause namedCause) {
         return this;
     }
 
+    public PhaseContext addCause(Cause cause) {
```

Comment:
What's the best way to make all events in the fake player phase have the plugin-provided cause? I figured the cause should just get added to the context.